pkgpanda/__init__.py

```diff
@@ -754,6 +797,20 @@ def symlink_all(src, dest):
                         uid = sysusers.get_uid(package.username)
                         check_call(['chown', '-R', str(uid), state_dir_path])
 
+            if package.sysctl:
+                service_names = _get_service_names(package.path)
+
+                if not service_names:
+                    raise ValueError("service name required for sysctl could not be determined for {package}".format(
+                            package=package.id))
+
+                for service in service_names:
+                    if service in package.sysctl:
+                        dcos_service_configuration["sysctl"][service] = package.sysctl[service]
+
+        new_dcos_service_json = self._make_abs("dcos-service-configuration.json.new")
```

Comment:
Can we just drop this in `/opt/mesosphere/etc` rather than having to do the manual swapping stuff here ourselves?